docker/k8s/vtadmin/Dockerfile

```diff
@@ -40,16 +39,29 @@ COPY --from=k8s /vt/web/vtadmin /vt/web/vtadmin
 RUN npm --prefix /vt/web/vtadmin ci && \
     npm run --prefix /vt/web/vtadmin build
 
-# add vitess user/group and add permissions
-RUN deluser node && \
-   groupadd -r --gid 2000 vitess && \
-   useradd -r -g vitess --uid 1000 vitess && \
-   chown -R vitess:vitess /vt && \
-   chown -R vitess:vitess /vtdataroot
 
-USER vitess
+FROM nginx:1.23.1-alpine AS nginx
```

Comment:
I recommend you to use official nginx unprivileged image it will avoid to re-write some logic.